src/metrics/print.rs

```diff
@@ -11,16 +11,13 @@ impl Consumer for Print {
         rx: &mut Receiver<Vec<String>>,
         column_names: Vec<String>,
     ) -> Result<(), String> {
-        println!(
-            "#\t{}\t{}\n------------------------------------------",
-            column_names[1], column_names[0]
-        );
+        println!("#,{}", column_names.join(","));
         let mut count = 1;
+
         while let Some(entry) = rx.recv().await {
-            println!("{}\t{}\t\t{}", count, &entry[1], &entry[0]);
+            println!("{},{}", count, entry.join(","));
```

Comment:
I feel like we should handle escaping for entries that contain `,`.  I bet there's a crate for writing csv files output, but I didn't find it with a super quick search.

Reply:
Oh, one other thing. We could make a "test producer" that supplies dummy data to test it!

In that case, it might be nice to also modify it so that we emit to any `Write` implementation? I realize now this is suboptimal since it's actually blocking I/O, so we should probably wrap these calls in `tokio::spawn_blocking`.